Sources/View/Widget/SelectionWidget.swift

```diff
@@ -95,8 +89,7 @@ final class SelectionWidget: AbstractWidget {
 
         tableView.items = field.fieldSelectionOptions ?? [HyperwalletFieldSelectionOption]()
         tableView.selectedHandler = { option in
-            self.labelField.text = option.label.localized()
-            self.selectedValue = option.value
+            self.updateLabelFieldValue(option)
```

Comment:
```self```

Reply:
We can't remove because is a closure scope